indexing-service/src/test/java/io/druid/indexing/overlord/TestRemoteTaskRunnerConfig.java

```diff
@@ -33,7 +33,8 @@ public Period getTaskAssignmentTimeout()
   @Override
   public long getMaxZnodeBytes()
   {
-    return 1000;
+    // make sure this is large enough, otherwise RemoteTaskRunnerTest might fail unexpectedly
+    return 10 * 1024;
```

Comment:
what this config does is that if a znode is greater than a certain size, an exception is thrown. It needs to be set 'large enough'